src/librustc/dep_graph/dep_node.rs

```diff
@@ -638,6 +638,7 @@ define_dep_nodes!( <'tcx>
     [input] TargetFeaturesWhitelist,
     [] TargetFeaturesEnabled(DefId),
 
+    [] InstanceDefSizeEstimate { instance_def: InstanceDef<'tcx> },
```

Comment:
Is `[]` correct here? I wasn't sure what this meant.

Reply:
Yes, that's correct. Leaving the brackets empty means that this is a regular query/dep-node.